.pre-commit-config.yaml

```
@@ -19,6 +19,16 @@ repos:
       - id: trailing-whitespace
         types_or: [c, python, rst]
 
+  - repo: local
+    hooks:
+      - id: c-file-whitespace
+        name: "Check C file whitespace"
+        entry: "python Tools/patchcheck/untabify.py"
+        language: "system"
+        types_or: ['c', 'c++']
+        # Don't check the style of copies directories containing external libraries
+        exclude: '^Modules/(_decimal/libmpdec|expat|zlib)/.*$'
```

The `Modules/zlib/` files were removed in https://github.com/python/cpython/pull/3375:

```suggestion
        exclude: '^Modules/(_decimal/libmpdec|expat)/.*$'
```

(I guess `os.path.join('Modules', 'zlib'),` wasn't found when grepping for `Modules/zlib`.)